blockchain/reactor.go

```diff
@@ -75,8 +76,9 @@ func NewBlockchainReactor(state sm.State, blockExec *sm.BlockExecutor, store *Bl
 			store.Height()))
 	}
 
-	const capacity = 1000 // must be bigger than peers count
-	requestsCh := make(chan BlockRequest, capacity)
+	requestsCh := make(chan BlockRequest, maxTotalRequesters)
```

Comment:
What is a benefit of this change?

Reply:
it's just some reasonable constant that is already defined.  vs having a magic 1000 in the middle of a function declaration.